### ISSUE TITLE
Requtest:  continuing a partially completed decryption process

### ISSUE THREAD

**Issue by FraserCorrance** [andlabs/reallymine#12]
Would it be possible to have Reallymine create a log file so that the process can be stopped and restarted from where it left off? This would be really handy. I recently had the power go out when I was decrypting a drive that was reading really slow (see previous post for more details). After 5 days of scanning I will now have to start all over.

Thank you and keep up the great work!

**Comment by andlabs** [comment]
I wonder how ddrescue does its log files in a reliable way...

**Comment by FraserCorrance** [comment]
I could send you a ddrescue log file if you think that might give you any clues. I am just a script kiddie. I am coding impaired.

**Comment by andlabs** [comment]
No, that alone won't help, but thanks for the offer. I'd have to look at the ddrescue source to see what it does.
